admob/Podfile

```diff
@@ -2,7 +2,7 @@ use_frameworks!
 platform :ios, '8.0'
 
 pod 'Firebase/Analytics'
-pod 'Firebase/AdMob'
+pod 'Google-Mobile-Ads-SDK'
```

Comment:
Since this is the Firebase quickstart, perhaps should still access the pod via the Firebase subspec?

Reply:
This change was specifically requested in b/148497211, though I am lacking the context behind why.

Reply:
Context cleared up in b/148497211. Thanks.